pages.ru/common/head.md

```diff
@@ -0,0 +1,20 @@
+# head
+
+> Output the first part of files.
```

Comment:
Wondering why this line is not in Russian ?

Reply:
Nice catch! Fixed.